gravitee-am-ui/src/app/domain/settings/audits/audit/audit.component.html

```diff
@@ -42,11 +42,16 @@ <h1>Audit log detail</h1>
           <input matInput placeholder="Event Type" disabled [value]="audit.type" />
           <mat-hint>Event type</mat-hint>
         </mat-form-field>
-        <mat-form-field *ngIf="audit.actor?.displayName" appearance="outline" floatLabel="always">
+        <mat-form-field *ngIf="audit.actor" appearance="outline" floatLabel="always">
           <mat-label>Actor</mat-label>
-          <input matInput placeholder="Actor" disabled [value]="audit.actor?.displayName" />
+          <input matInput placeholder="Actor" disabled [value]="getActorDetails()" />
           <mat-hint>Who triggered the event</mat-hint>
         </mat-form-field>
+        <mat-form-field *ngIf="audit.target" appearance="outline" floatLabel="always">
+          <mat-label>Target</mat-label>
+          <input matInput placeholder="Target" disabled [value]="getTargetDetails()" />
+          <mat-hint>The target object</mat-hint>
+        </mat-form-field>
```

Comment:
TODO: can you please keep only the displayName as it was before ? The change you propose is not really readable so I would prefer to provide the display name only.

Reply:
fixed